tracy-client-sys/build.rs

```diff
@@ -48,25 +48,32 @@ fn set_feature_defines(mut c: cc::Build) -> cc::Build {
 }
 
 fn main() {
-    if std::env::var_os("CARGO_FEATURE_ENABLE").is_some() {
-        set_feature_defines(cc::Build::new())
-            .file("tracy/TracyClient.cpp")
-            .define("TRACY_MANUAL_LIFETIME", None)
-            .define("TRACY_DELAYED_INIT", None)
-            .warnings(false)
-            .cpp(true)
-            .flag_if_supported("-std=c++11")
-            .compile("libtracy-client.a");
-    }
-
-    match std::env::var("CARGO_CFG_TARGET_OS") {
-        Ok(target_os) => {
-            link_libraries(&target_os);
+    if let Ok(lib) = std::env::var("TRACY_CLIENT_LIB") {
```

Comment:
Hm, while being able to specify this is nice, I wonder if we can do something to not require specifying `tracy-release` or something here if people just run `make` and don’t rename the client libraries?

I’d argue that we should at the very least have an enabled-by-default crate feature for use of `vendor`ed version of tracy.

Reply:
I'm not sure what you mean with `tracy-release` but my intention was to not change the default behaviour so only if you specify `TRACY_CLIENT_LIB` the bundled version is not compiled.

Reply:
Well, if you want to use `TRACY_CLIENT_LIB` you must now specify the library name (the `{name}` part of `lib{name}.{so,dylib,a}` on unixes), but the tracy’s makefiles produce a client library with a well-known name. 

For cmake build that seems to be `libTracyClient.a` and the makefiles appear to spit out a `libtracy-release.so`/`libtracy-debug.so`. I guess its okay to keep this as is in that case…